packages/rs-sdk/src/sdk.rs

```diff
@@ -665,6 +670,28 @@ impl SdkBuilder {
         self
     }
 
+    /// Configure CA certificate to use when verifying TLS connections.
+    ///
+    /// Used mainly for testing purposes and local networks.
+    ///
+    /// If not set, uses standard system CA certificates.
+    pub fn with_ca_certificate(mut self, pem_certificate: &[u8]) -> Self {
+        self.ca_certificate = Some(pem_certificate.to_vec());
+        self
+    }
+
+    /// Load CA certificate from file.
+    ///
+    /// This is a convenience method that reads the certificate from a file and sets it using
+    /// [SdkBuilder::with_ca_certificate()].
+    pub fn with_ca_certificate_file(
```

Comment:
I think a better approach would be to have just a single `with_ca_certificate` and expect an actual certificate value, users can figure out where and how to get it as long as it can be constructed

Reply:
refactored `with_ca_certificate(mut self, pem_certificate: Certificate)`, keeping `with_ca_certificate_file` as a conveniance method for users.